docs/source/release_notes.rst

```diff
@@ -5,11 +5,20 @@ Release Notes
 |
 
 **Future Release**
+    * Enhancements
+    * Fixes
+    * Documentation Changes
+    * Testing Changes
+
+    | Thanks to the following people for contributing to this release:
```

Comment:
Do we want the Future Release section to be visible?

Reply:
I commented out like in Featuretools